src/currency.h

```diff
@@ -60,6 +60,9 @@ enum Currencies {
 	CURRENCY_IRR,       ///< Iranian Rial
 	CURRENCY_RUB,       ///< New Russian Ruble
 	CURRENCY_MXN,       ///< Mexican Peso
+	CURRENCY_NTD,       ///< New Taiwan Dollar
+	CURRENCY_CNY,       ///< Chinese Renminbi
+	CURRENCY_HKD,       ///< Hong Hong Dollar
```

Comment:
typo in the comment here: "Hong Hong Dollar" -> "Hong Kong Dollar"

Reply:
Thanks.